README.md

```diff
@@ -138,3 +121,12 @@ Here are some resources actively maintained by the Beam community to help you ge
   </tr>
 </tbody>
 </table>
+
+## Contact Us
+
+To get involved with Apache Beam:
+
+* Subscribe or e-mail the [user@beam.apache.org](http://mail-archives.apache.org/mod_mbox/beam-user/) list.
+* Subscribe or e-mail the [dev@beam.apache.org](http://mail-archives.apache.org/mod_mbox/beam-dev/) list.
```

Comment:
Could we link the subscribe pieces to https://beam.apache.org/community/contact-us/#:~:text=Subscribe%20and%20Unsubscribe